### PR TITLE
Fix several typos in the French translation

### PR DESCRIPTION
**Pull request by trebmuh**
HTH

### CODE REVIEW

File: res/i18n/fr/labels.json [hunk 1]
```diff
@@ -371,7 +371,7 @@
 	"ratio": "Ratio",
 	"ratio_high": "Ratio haut",
 	"ratio_low": "Ratio bas",
-	"ratio_knee:db": "Ratio,\genoux (dB)",
+	"ratio_knee:db": "Ratio,\genou (dB)",
```
Comment (sadko4u) [inline res/i18n/fr/labels.json:374]: Here we get propably improper new line escape sequence, please fix this with '\n'.

Comment (trebmuh) [reply res/i18n/fr/labels.json]: done

**Comment by sadko4u** [comment]
Merged changes into **devel** branch: 7cd37b24

**Comment by sadko4u** [comment]
Thanks

**Comment by sadko4u** [comment]
Available in 1.1.26 release

**Pull request merged**